include/roaring/portability.h

```diff
@@ -27,11 +27,10 @@
 #include <malloc.h> // this should never be needed but there are some reports that it is needed.
 #endif
 
-#if __SIZEOF_LONG_LONG__ != 8
+#if defined(__SIZEOF_LONG_LONG__) && __SIZEOF_LONG_LONG__ != 8
```

Comment:
@ksajme Interesting issue. Looks like we do not need to check that long long is 8 bytes in Visual Studio, even when compiling a 32-bit binary.